### DISCUSSION TITLE
Ayuda con el proceso en misión backend

### DISCUSSION THREAD

**Discussion by musicod3r** [Launch-X-Latam/MisionFrontEnd#513] [Team Posts]
Hola a todos, apenas acabo de ver que estaba dentro del programa, veo que viene misión backend, pero estoy desubicado, como se ingresa a la plataforma para ver los videos? hay clases sincrónicas? que días son? las clases de misión frontend las grabaron? ya revisé los correos y no hay nada que me envíe a la plataforma de estudio

**Comment by dzarkV** [comment]
Hola, puedes encontrar algunos recursos [aquí](https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/39/comments/1).
Ánimo que se puede.

**Comment by musicod3r** [comment]
Muchas gracias compañero
